src/cassandra/custom_payload.rs

```diff
@@ -1,7 +1,56 @@
-// use cassandra_sys::cass_custom_payload_free;
-// use cassandra_sys::cass_custom_payload_new;
-// use cassandra_sys::cass_custom_payload_set;
+use crate::cassandra::error::*;
+use crate::cassandra::util::Protected;
 
-// use cassandra_sys::CassCustomPayload as _CassCustomPayload;
-//
-// pub struct CustomPayload(_CasCustomPayload);
+use crate::cassandra_sys::cass_custom_payload_free;
+use crate::cassandra_sys::cass_custom_payload_new;
+use crate::cassandra_sys::cass_custom_payload_set;
+use crate::cassandra_sys::cass_custom_payload_set_n;
+use crate::cassandra_sys::CassCustomPayload as _CassCustomPayload;
+use std::collections::HashMap;
+use std::os::raw::c_char;
+
+pub type CustomPayloadResponse = HashMap<String, Vec<u8>>;
+
+/// Custom payloads not fully supported yet
```

Comment:
Is this comment still true?

Reply:
Great question - I don't think this adds support for adding custom payloads to queries yet, so yes I think it is still true.